maintenance_github_grooming.pt.Rmd

```diff
@@ -0,0 +1,48 @@
+---
+aliases: grooming.html
+---
+
+# GitHub Grooming {#grooming}
+
+```{block, type="summaryblock"}
+Atualmente, os pacotes do rOpenSci são, em sua grande maioria, desenvolvidos no GitHub. Aqui estão algumas dicas para aproveitar a plataforma em uma seção sobre [tornar seu repositório mais detectável] (#repodiscoverability) e uma seção sobre [comercializar sua própria conta do GitHub após passar pela revisão por pares] (#marketown).
+
+```
+
+## Torne seu repositório mais detectável {#repodiscoverability}
+
+### Tópicos do repositório do GitHub {#git-hub-repo-topics}
+
+GitHub [tópicos do repositório](https://blog.github.com/2017-01-31-introducing-topics/) ajudam a navegar e pesquisar repositórios do GitHub, são usados por [R-universe em páginas de pacotes e para resultados de pesquisa](https://github.com/r-universe-org/help#how-to-add-keyword-labels-to-an-r-package) e são digeridos por [`codemetar`](https://github.com/ropensci/codemetar) para palavras-chave do registro rOpenSci.
+
+Recomendamos que você:
+
+- Adicionar "r", "r-package" e "rstats" como tópicos ao seu repositório de pacotes.
+
+- Adicionar quaisquer outros tópicos relevantes ao repositório de pacotes que você possui.
```

Comment:
I assume it is 1 repository per package. Or am I wrong and it is common in R environment to have more than one package per repo?

```suggestion
- Adicionar "r", "r-package" e "rstats" como tópicos ao seu repositório de pacote.

- Adicionar quaisquer outros tópicos relevantes ao repositório de pacote que você possui.
```

Reply:
It is correct, it is one repo per package.